patient-db/src/main/java/com/hartwig/hmftools/patientdb/clinical/curators/DoidNodesResolver.java

```diff
@@ -0,0 +1,32 @@
+package com.hartwig.hmftools.patientdb.clinical.curators;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.Lists;
+import com.hartwig.hmftools.common.doid.DoidNode;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+
+public class DoidNodesResolver {
+
+    @NotNull
+    private final List<DoidNode> doidNodes;
+
+    public DoidNodesResolver(@NotNull List<DoidNode> doidNodes) {
+        this.doidNodes = doidNodes;
+    }
+
+    @NotNull
+    @VisibleForTesting
```

Comment:
Is this `VisibleForTesting` annotation still relevant? Because it seems to also be used in the `LimsPatientReader` which isn't a test class

Reply:
Agreed, done.